docs/current/sdk/go/snippets/step2/main.go

```diff
@@ -0,0 +1,33 @@
+package main
```

Comment:
Considering that we will be adding many more such snippets, WDYT of changing this file structure a bit? A separate directory related to each guide, with all the code for that guide within it.
snippets/guide-name/script.go

eg. snippets/get-started/main-step2.go, etc.

In future, we would have

eg. snippets/host-api/example1.go, snippets/container-api/example5.go, etc

Reply:
Please feel free to do as you think is best!

I agree with your suggestion, what I did here was really quick & dirty just so I could get go build running on them

Reply:
Oh just one thing:

`snippets/get-started/main-step2.go` --> this unfortunately won't work because we can't have more than one `main` function per package

So if we do that, it should be `snippets/get-started/step2/main.go` or something